lib/jsdom/living/nodes/HTMLElement-impl.js

```diff
@@ -46,21 +47,17 @@ class HTMLElementImpl extends ElementImpl {
       return;
     }
 
-    const event = MouseEvent.createImpl([
-      "click",
-      {
-        bubbles: true,
-        cancelable: true,
-        composed: true,
-        view: this.ownerDocument.defaultView
-      }
-    ], {});
-
     // Run synthetic click activation steps. According to the spec,
     // this should not be calling dispatchEvent, but it matches browser behavior.
     // See: https://www.w3.org/Bugs/Public/show_bug.cgi?id=12230
     // See also: https://github.com/whatwg/html/issues/805
```

Comment:
The spec has since been changed to handle this, so this comment is no longer accurate. Probably best to do any changes here in a followup though.

(For example, this seems buggy in that disabled elements will have _clickInProgress forever! We should write a test to show that bug and then fix it. Also the comment in click()'s header says we have no implementation of isTrusted, heh.)